PhysicsTools/PatAlgos/python/slimming/puppiForMET_cff.py

```diff
@@ -18,25 +18,26 @@ def makePuppies( process ):
 
 
     process.puppiNoLep = process.puppi.clone()
-    process.puppiNoLep.candName = cms.InputTag('pfNoLepPUPPI') 
-
-    process.puppiMerged = cms.EDProducer("CandViewMerger",src = cms.VInputTag( 'puppiNoLep','pfLeptonsPUPPET'))
+    process.puppiNoLep.candName = cms.InputTag('particleFlow')
+    process.puppiNoLep.useWeightsNoLep = cms.bool(True)
+    
+#    process.puppiMerged = cms.EDProducer("CandViewMerger",src = cms.VInputTag( 'puppiNoLep','pfLeptonsPUPPET'))
 
 ## puppi met
     process.puppiForMET = cms.EDProducer("PuppiPhoton",
-                                         candName       = cms.InputTag('packedPFCandidates'),
-                                         puppiCandName  = cms.InputTag('puppi'),
-                                         photonName     = cms.InputTag('slimmedPhotons'),
-                                         photonId       = cms.InputTag("egmPhotonIDs:cutBasedPhotonID_PHYS14_PU20bx25_V2p1-standalone-loose"),
-                                         pt             = cms.double(10),
-                                         useRefs        = cms.bool(True),
-                                         dRMatch        = cms.vdouble(10,10,10,10),
-                                         pdgids         = cms.vint32 (22,11,211,130),
-                                         weight         = cms.double(1.),
-                                         useValueMap    = cms.bool(False),
-                                         weightsName    = cms.InputTag('puppi'),
+      candName = cms.InputTag("particleFlow"),
+                                         dRMatch = cms.vdouble(10, 10, 10, 10),
+                                         pdgids = cms.vint32(22, 11, 211, 130),
+                                         photonId = cms.InputTag("egmPhotonIDs","cutBasedPhotonID_PHYS14_PU20bx25_V2p1-standalone-loose"),
+                                         photonName = cms.InputTag("slimmedPhotons"),
+                                         pt = cms.double(10),
+                                         puppiCandName = cms.InputTag("puppi"),
+                                         useRefs = cms.bool(True),
+                                         useValueMap = cms.bool(False),
+                                         weight = cms.double(1.0),
+                                         weightsName = cms.InputTag("puppiNoLep")
```

Comment:
it would be easier to follow what has changed if parameters were not reshuffled and reindented